core/services/wifi/WifiManager.py

```diff
@@ -28,6 +30,7 @@ def connect(self, path: Any) -> None:
         self._updated_scan_results: Optional[List[ScannedWifiNetwork]] = None
         self._ignored_reconnection_networks: List[str] = []
         self.connection_status = ConnectionStatus.UNKNOWN
+        self._hotspot = HotspotManager("wlan0", IPv4Address("192.168.3.2"))
```

Comment:
I would recommend to move to `192.168.*133*.2`or something not close to the lower numbers, with that, we can avoid conflict with user network configuration inside the ROV in the future. @Williangalvani what you think ?

Reply:
> I would recommend to move to `192.168.*133*.2`or something not close to the lower numbers, with that, we can avoid conflict with user network configuration inside the ROV in the future. @Williangalvani what you think ?

Usually, above 100 is used for dynamic IPs, so it might clash and get rejected.

Reply:
Why don't we ask for an IP? Wouldn't it be easier?

Reply:
> Why don't we ask for an IP? Wouldn't it be easier?

Hotspot should be something enabled by default. What we could add is an option to change the IP. But to do that, it may be necessary to change /etc/dhcpcd.conf file, and that's a pain..

Reply:
Is there a reason not to end with .1? we only do .2 right now for... historical reasons. We should move away from this as we can.

As for the 192.168.3.* part, the only issue I can think of is that a user could have the same thought process and also setup a networking at (.2 +1), but I don't think it is likely for an user to have two different network

Reply:
> As for the 192.168.3.* part, the only issue I can think of is that a user could have the same thought process and also setup a networking at (.2 +1), but I don't think it is likely for an user to have two different networks in the same rov


for now, I would also go forward @joaoantoniocardoso previous comment:


> Usually, above 100 is used for dynamic IPs, so it might clash and get rejected.

Reply:
So `192.168.42.1`?

Reply:
Looks good.